articles/how-to-build-a-clock-with-javascript-and-svg/index.md

```diff
@@ -0,0 +1,486 @@
+
+  ```
+
+- SVG Stroking
+
+  ```svg
+  <svg height="80" width="300">
+    <g fill="none">
+    <path stroke="red" d="M5 20 l215 0" />
+    <path stroke="black" d="M5 40 l215 0" />
+    <path stroke="blue" d="M5 60 l215 0" />
+    </g>
+  </svg>
+
+  ```
+
+- SVG Blur Effects
+  ```svg
+        <svg height="110" width="110">
+            <defs>
+                <filter id="f1" x="0" y="0">
+                <feGaussianBlur in="SourceGraphic" stdDeviation="15" />
+                </filter>
+            </defs>
+            <rect width="90" height="90" stroke="green" stroke-width="3"
+            fill="yellow" filter="url(#f1)" />
+        </svg>
+  ```
+- SVG Drop Shadows
+  ```svg
+        <svg height="120" width="120">
+            <defs>
+                <filter id="f1" x="0" y="0" width="200%" height="200%">
+                    <feOffset result="offOut" in="SourceGraphic" dx="20" dy="20" />
+                    <feBlend in="SourceGraphic" in2="offOut" mode="normal" />
+                </filter>
+            </defs>
+            <rect width="90" height="90" stroke="green" stroke-width="3"
+        fill="yellow" filter="url(#f1)" />
+        </svg>
+  ```
+- SVG Linear
+  ```svg
+        <svg height="150" width="400">
+            <defs>
+                <linearGradient id="grad1" x1="0%" y1="0%" x2="100%" y2="0%">
+                <stop offset="0%" style="stop-color:rgb(255,255,0);stop-opacity:1" />
+                <stop offset="100%" style="stop-color:rgb(255,0,0);stop-opacity:1" />
+                </linearGradient>
+            </defs>
+            <ellipse cx="200" cy="70" rx="85" ry="55" fill="url(#grad1)" />
+        </svg>
+  ```
+- SVG Radial
+  ```svg
+    <svg height="150" width="500">
+        <defs>
+            <radialGradient id="grad1" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
+            <stop offset="0%" style="stop-color:rgb(255,255,255);
+            stop-opacity:0" />
+            <stop offset="100%" style="stop-color:rgb(0,0,255);stop-opacity:1" />
+            </radialGradient>
+        </defs>
+        <ellipse cx="200" cy="70" rx="85" ry="55" fill="url(#grad1)" />
+    </svg>
+  ```
+
+### Benefits of SVG
+
+The following are some of the benefits of using SVG over other image formats (such as JPEG and GIF):
+
+- Any text editor can be used to generate and edit SVG files.
+- Photos in SVG format can be found, indexed, scripted, and compressed.
+- SVG images can be scaled to any size.
+- SVG images can be printed in high resolution at any scale.
+- Images in SVG can be zoomed in and out.
+- When SVG graphics are zoomed or resized, they do not lose quality.
+- SVG is a free and open format.
+- SVG files are made up entirely of XML.
+
+### How to implement JavaScript with SVG on the web
+
+Now, using JavaScript and one of the SVG examples from above, let's create a simple webpage to develop a clock on the web.
+
+**Step 1:** Navigate to any Directory of your choice on your pc, in the terminal, type:
+
+```Bash
```

Comment:
Inform the user that `code .` will not work if they do not have visual studio code installed

Reply:
Done